wordpress-mobile/WordPress-Android
# Add support for social webauthn/passkey

This PR serves as the WordPress companion, adding support for the updated login functionality which relates to social login integration with security key. The base PRs are as follows:

- https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2894
- https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/128

### Merge Instructions
- Make sure the two PRs above have been merged
- Grab the build hashes and update in this PR
- Remove Do Not Merge label
- Merge as normal

## To Test:
- Build a WP signed APK locally (social login can not be tested with a debug version)
- Login with a Google account associated with WordPress.com using the Login with Google button during the login flow. (make sure this same account also has a Security key configured for your device)
- Verify that the 2FA view is opened and the use a security key option is available.
- Continue the usual Security key flow and make sure everything works as expected.

-----

## Regression Notes

1. Potential unintended areas of impact
The security key option does not work for social logins

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

-----

## PR Submission Checklist:

-----

## UI Changes Testing Checklist: N/A
